src/registrar/views/index.py

```diff
@@ -9,4 +10,13 @@ def index(request):
     if request.user.is_authenticated:
         applications = DomainApplication.objects.filter(creator=request.user)
         context["domain_applications"] = applications
+
+        domains = request.user.permissions.values(
+            "role",
+            pk=F("domain__id"),
+            name=F("domain__name"),
+            created_time=F("domain__created_at"),
+            application_status=F("domain__domain_application__status"),
+        )
+        context["domains"] = domains
```

Comment:
OKAY, the comment I left on this code which was more erudite before GitHub erased it for me was that I'd love to see this done in a less opt-in way, possibly with models.Manager or models.QuerySet.as_manager(), such that calling `Domain.objects.all()` (or the like) would return `[]` unless you were a superuser. BUT that this is fine as-is if you don't have time for that right now.